fire/cli/mtl.py

```diff
@@ -179,7 +180,7 @@ def hent_sridid(db, srid: str) -> int:
 
 
 # ------------------------------------------------------------------------------
-def path_to_origin(graph, start, origin, path=[]):
+def find_path(graph: Dict[str, Set[str]], start: str, end: str, path: List[str] = []):
```

Comment:
`origin` ser ud til at være forsvundet her. Den bliver i hvert fald brugt i koden nedenfor men tilsyneladende ikke defineret nogle steder. 

Kunne godt se ud til at være sket i forbindelse med et rebase.

Reply:
@busstoptaktik i øvrigt ikke særligt betryggende at test-suiten ikke fejler her - kan det passe? Jeg er klar over at der ikke (endnu) er en dedikeret test til denne funktion men jeg ville alligevel tro at den blev motioneret af den test-case der er sat ind i øjeblikket.

Reply:
Skarpt set, men jeg tror problemet er at argumentet har skiftet navn fra origin til "end", så den stadig bør løbe korrekt igennem

> jeg ville alligevel tro at den blev motioneret af den test-case der er sat ind i øjeblikket.

Det bliver den, i den forstand at den bliver kaldt, og at den burde protestere hvis den bliver kaldt med forkerte typer eller forkert antal argumenter - men den kaldes korrekt.

Testen er p.t. rudimentær og bare anslag til noget mere holdbart. Blandt andet fordi testcasen endnu er mistænkelig, numerisk set.

Reply:
Master ser lige nu sådan her ud:

https://github.com/Kortforsyningen/FIRE/blob/612abd2f2e7a02338d8bb248cc09413b3a481abe/fire/cli/mtl.py#L182-L199

Funktionen blev ændret her: https://github.com/Kortforsyningen/FIRE/commit/cc1ff355ba600a2bd8af0c241c1013f804683ccc

Så, jeg holder fast - der er gået kuk i et rebase, og der er noget mystisk i test-suiten.

Reply:
Testen overlever fordi funktionen kaldes med unavngivne parametre, og rækkefølgen på input er den samme som den hele tiden har været.